test/functional/rpc_invalidateblock.py

```diff
@@ -83,6 +84,11 @@ def run_test(self):
         # Should be back at the tip by now
         assert_equal(self.nodes[1].getbestblockhash(), blocks[-1])
 
+        self.log.info("Verify that invalidating an unknown block throws an error")
+        chain_tips = self.nodes[1].getchaintips()
+        assert_raises_rpc_error(-5, "Block not found", self.nodes[1].invalidateblock, "00" * 32)
+        assert_equal(chain_tips, self.nodes[1].getchaintips())
+
```

Comment:
This fails: https://cirrus-ci.com/task/4697486185988096?logs=ci#L5532

Reply:
I'm gonna fix it.

Reply:
https://github.com/bitcoin/bitcoin/pull/26054